src/gameplay/GameStart.js

```diff
@@ -101,8 +105,8 @@ function GameStart(props) {
 	        <li className="list-group-item" key={name}>{name} is in the lobby</li>
 	      ))}
 	    </ol>
-      <div align="center"> <h4>Current Room: {props.roomName}</h4> </div>
-	    <JoinGame isHost={props.isHost} roomName={props.roomName} playerID = {props.playerID} setPlayerIndex = {props.setPlayerIndex} setPlayerNames = {props.setPlayerNames}/>
+      <div align="center"> <h4>Current Room: {roomName}</h4> </div>
+	    <JoinGame isHost={props.isHost} roomName={roomName} playerID = {props.playerID} setPlayerIndex = {props.setPlayerIndex} setPlayerNames = {props.setPlayerNames}/>
```

Comment:
here

Reply:
oh ya here I can not pass it in and use context too, I just missed this one

Reply:
thought so too, cool

Reply:
either way works, actually I think I left it cause this component is held within GameStart component so passing it as a prop made sense, the other component is in another file so I used context there. TBH either works tho